### PR TITLE
BAH 8940: Document Type dropdown display during upload

### PR DESCRIPTION
**Pull request by robfelder**
## Description
This PR addresses the need to update the document type functionality for uploading a completed 8940 form. This updates it to use the reusable form component and default to a specific document 
type.

Link to story: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/14190

## Testing done
Manual & Unit

## Screenshots
screen shot 2019-01-02 at 11 28 45 am

## Acceptance criteria
- [x] During the interview if the user specifically selects to upload a completed 8940 then the document type list will automatically default to the Form name (VA 21-8940-Veterans Application for Increased Compensation Based on Unemployability)
- [x] In the scenario where the Document Type is defaulted to a specific form then it is not editable by the user
- [x] Document name field will not display to the user at all (remove it)
- [x] During the 8940 interview if the user is uploading a general supporting document (not a specific form) then the document type list will be filtered and displayed to the user based on the options listed on the attached spreadsheet/tab _8940_in the VBMS Description Text column
- [x] Document name field will not display to the user at all (remove it)
- [x] User is able to delete the file if needed
- [x] User can add another document
- [x] User can continue to next step in the flow

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/incidentUnitAssignment.jsx [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 
-export const ptsdAssignmentDescription = <h5>Your assignment</h5>;
+export const ptsdAssignmentDescription = (
+  <div aria-valuetext="Your assignment">
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/content/incidentUnitAssignment.jsx]: This resolves the 508 title defect?

**Review by aaronjlech — APPROVED** [review]
This looks good. Just one small question about the titles, also I have the updates coming in for the attachments, so depending on the order our PRs go, I can update this page, or you can make the updated changes based off of what there is for 781.

**Comment by imeecuison** [comment]
@aaronjlech : This is ready for review again. I fixed the upload issue. The dropdown is no longer showing and the file name is also not showing. Thank you!

File: src/applications/disability-benefits/all-claims/sass/disability-benefits.scss [hunk 2]
```diff
@@ -1,13 +1,13 @@
 // Variables
 
-@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/sass/disability-benefits.scss]: Can we change all of this back, looks like your editor may have gotten a little opinionated.

Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/sass/disability-benefits.scss]: @imeecuison ^^^^^^^^^^^^^

Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/sass/disability-benefits.scss]: yikes! okay!

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group - Could I get a code review on this at your earliest convenience? Thanks!

**Comment by rtluu** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group would you all be able to give this a  👀 ? Thanks in advance!

File: src/applications/disability-benefits/all-claims/content/incidentDate.jsx [hunk 3]
```diff
@@ -17,7 +17,9 @@ const eventHelpText = (
 
 export const ptsdDateDescription = (
   <div>
-    <h5>Approximate event date</h5>
+    <div aria-valuetext="Approximate event date">
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/content/incidentDate.jsx]: What is this aria attribute for? It does not look like a valid attribute for a plain div (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute)

Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/content/incidentDate.jsx]: This resolves the 508 title defect?

**Review by jbalboni — APPROVED** [review]

**Pull request merged**